demos/shapes-custom-question-authoring-demo/src/question/index.js

```diff
@@ -0,0 +1,339 @@
+
+
+        };
+        facade.enable = () => {
+            /**
+             * The purpose of this method is to re-enable learner interaction with your question's UI
+             * after it has been previously disabled.
+             *               
+             * (For example, you plan to temporarily disable the question UI for a student taking the assessment until they complete another task like spend a set time reading the instructions.)
+             */
+
+            const shapeButtons = Array.from(el.querySelectorAll('.shape-wrapper button'));
+            shapeButtons.forEach(btn => btn.removeAttribute('disabled'));
+        };
+        facade.resetResponse = () => {
+            /**
+             * The purpose of this method is to support removal of a previously entered response to your custom question
+             * and restoring the question to its initial blank state before a response was entered.
+             * (For example, in a multiple choice question where a student has picked a choice, you reset the question so that no choices are picked at all.)
+             */
+
+            // TODO: Requires implementation, you could use the following steps
+
+            // trigger a 'resetResponse' event to reset the value of response
+            // this.events.trigger('resetResponse');
+
+            // reset other states if you need
+
+            // re-render the component, manage the 'reset' state by yourself
+        };
+        facade.showValidationUI = () => {
+            // TODO: requires implementation
+            /**
+             * The purpose of this method is to update your custom question's UI with visual feedback 
+             * after the learner presses the Check Answer button.
+             * 
+             * The following is an example implementation that shows the standard Learnosity "checkmark / tick" for a correct answer
+             * or the standard Learnosity "x mark / cross" for an incorrect answer. 
+             */
+            
+            // EXAMPLE implemetation:
+
+            const answerIsCorrect = facade.isValid();
+            /**
+             * update the UI based on the result
+             * IMPORTANT: In both cases we need to target the element with the classname lrn_response_input
+             */
+            if (answerIsCorrect) {
+                /** if the answer is correct we will add the Learnosity utility class lrn_correct, which will automatically show the Learnosity
+                 * checkmark (tick) you are used to on regular Learnosity quesiton types
+                 */
+                el.querySelector(".lrn_response_input").classList.add("lrn_correct");
+            } else {
+                /** if the answer is incorrect we will add the Learnosity utility class lrn_incorrect, which will automatically show the Learnosity
+                 * x (cross) you are used to on regular Learnosity quesiton types
+                 */
```

Comment:
- Fix comments for readability
- Fix typo question
```suggestion
                /** 
                 * if the answer is incorrect we will add the Learnosity utility class lrn_incorrect,
                 * which will automatically show the Learnosity
                 * x (cross) you are used to on regular Learnosity quesiton types
                 **/
```